eth2deposit/intl/en/cli/generate_keys.json

```diff
@@ -0,0 +1,35 @@
+{
+    "validate_eth1_withdrawal_address": {
+        "err_invalid_ECDSA_hex_addr": "The given Eth1 address is not in hexadecimal encoded form.",
+        "msg_ECDSA_addr_withdrawal": "**[Warning] you are setting an Eth1 address as your withdrawal address. Please ensure that you have control over this address.**"
+    },
+    "generate_keys_arguments_decorator": {
+        "num_validators": {
+            "help": "The number of validators keys you want to generate (you can always generate more later)",
+            "prompt": "Please choose how many validators you wish to run"
+        },
+        "folder": {
+            "help": "The number of validators keys you want to generate (you can always generate more later)"
```

Comment:
Aargh, I can't believe I missed this! It is correct on the master branch, it was just a copy-pasta mistake I made when extracting the text to json. That said, it it *only* in the help file, so no major loss, I will fix the English one and remove the other translations (so it will revert to explaining this in English when `--help` is requested).